zstackctl/zstackctl/ctl.py

```
@@ -250,7 +274,26 @@
 configure_yum_repo_script = '''
 if [ -f /etc/redhat-release ]; then
     os_release=`cat /etc/redhat-release`
-    if [[ $os_release =~ ' 7' ]]; then
+    if [[ $os_release =~ 'Alibaba Cloud Linux' ]] && grep -Eq '^VERSION_ID="?4(\.|")' /etc/os-release 2>/dev/null; then
```

Ruff (0.15.12)

[warning] Invalid escape sequence: `\.`

Use a raw string literal

(W605)